baybe/utils/chemistry.py

```diff
@@ -26,8 +27,16 @@
 
 
 # Caching
-_cachedir = Path.home() / ".baybe_cache"
-_memory_utils = Memory(_cachedir / "utils")
+_cachedir = os.environ.get(
+    "BAYBE_CACHE_DIR", str(Path(tempfile.gettempdir()) / ".baybe_cache")
+)
+
+
+def _dummy_wrapper(func):
```

Comment:
Why is this wrapper necessary?

Reply:
if caching is turned off we need a wrapper that does nothing in line 39, hence this identity dummy

Ive also tried to do it more elegantly as lambda but i) lint complains and ii) tests failed